src/systems/systems/src/pressurization/mod.rs

```diff
@@ -106,6 +224,114 @@ impl Default for Pressurization {
     }
 }
 
+pub struct PressurizationOverheadPanel {
+    mode_sel: AutoManFaultPushButton,
+    man_vs_ctl_switch: SpringLoadedSwitch,
+    ldg_elev_knob: ValueKnob,
+    ditching: NormalOnPushButton,
+}
+
+impl PressurizationOverheadPanel {
+    pub fn new() -> Self {
+        Self {
+            mode_sel: AutoManFaultPushButton::new_auto("PRESS_MODE_SEL"),
+            man_vs_ctl_switch: SpringLoadedSwitch::new("PRESS_MAN_VS_CTL"),
+            ldg_elev_knob: ValueKnob::new_with_value("PRESS_LDG_ELEV", -2000.),
+            ditching: NormalOnPushButton::new_normal("PRESS_DITCHING"),
+        }
+    }
+
+    fn ldg_elev_is_auto(&self) -> bool {
+        let margin = 100.;
+        (self.ldg_elev_knob.value() + 2000.).abs() < margin
+    }
+
+    fn is_in_man_mode(&self) -> bool {
+        !self.mode_sel.is_auto()
+    }
+
+    fn man_vs_switch_position(&self) -> usize {
+        self.man_vs_ctl_switch.toggle_position()
+    }
+}
+
+impl SimulationElement for PressurizationOverheadPanel {
+    fn accept<T: SimulationElementVisitor>(&mut self, visitor: &mut T) {
+        self.mode_sel.accept(visitor);
+        self.man_vs_ctl_switch.accept(visitor);
+        self.ldg_elev_knob.accept(visitor);
+        self.ditching.accept(visitor);
+
+        visitor.visit(self);
+    }
+}
+impl Default for PressurizationOverheadPanel {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl ControllerSignal<PressureValveSignal> for PressurizationOverheadPanel {
+    fn signal(&self) -> Option<PressureValveSignal> {
+        if !self.is_in_man_mode() {
+            None
+        } else {
+            match self.man_vs_switch_position() {
+                0 => Some(PressureValveSignal::Open),
+                2 => Some(PressureValveSignal::Close),
+                _ => Some(PressureValveSignal::Neutral),
```

Comment:
**suggestion**: Of course catching any value but `0` and `2` and then returning `Some(PressureValveSignal::Neutral)` is possible. But having any other value than 1 for `PressureValveSignal::Neutral` might indicate a programming error. Maybe consider panicking?

Reply:
Added `panic!` for all cases not being 0, 1 or 2.